Formula/u-boot-tools.rb

```diff
@@ -26,8 +26,9 @@ def install
     # Replace keyword not present in make 3.81
     inreplace "Makefile", "undefine MK_ARCH", "unexport MK_ARCH"
 
-    system "make", "sandbox_defconfig"
-    system "make", "tools", "NO_SDL=1"
+    ENV["SOURCE_DATE_EPOCH"] = ""
+    system "make", "tools-only_defconfig"
+    system "make", "tools-only", "NO_SDL=1"
```

Comment:
How are these targets different from the ones we used to build?

Reply:
this is what it says in their documentation: [Das U-Boot](https://u-boot.readthedocs.io/en/latest/build/tools.html#building-tools-for-linux)

Reply:
Those docs don't seem to explain the difference. Did I just miss it?

Reply:
Sorry, no there is no explanation. I think there is no difference if it is only about building the host tools. The `sandbox` target defines a complete u-boot board definition while the `tools-only` target defines only a minimal configuration for just building the host tools.